app/src/main/res/layout/view_actors.xml

```diff
@@ -6,89 +6,89 @@
     android:background="@color/movie_background">
 
     <ImageView
-        android:id="@+id/imActor1"
+        android:id="@+id/im_robert"
```

Comment:
Rename with camel_case

Reply:
What if I decide to put another actor in thee first place?